src/execution/operator/join/physical_nested_loop_join.cpp

```diff
@@ -309,7 +315,10 @@ void PhysicalNestedLoopJoin::ResolveSimpleJoin(ExecutionContext &context, DataCh
 	state.left_condition.Reset();
 	state.lhs_executor.Execute(input, state.left_condition);
 
-	bool found_match[STANDARD_VECTOR_SIZE] = {false};
+	bool found_match[STANDARD_VECTOR_SIZE] = {true};
```

Comment:
This change seems off to me - is this made intentionally?

Reply:
No not made intentionally. Belonged to a previous solution which I ended up abandoning because I didn't really want to change this logic like this